llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7324,6 +7325,11 @@ DenseMap<const SCEV *, Value *> LoopVectorizationPlanner::executePlan(
          "Trying to execute plan with unsupported VF");
   assert(BestVPlan.hasUF(BestUF) &&
          "Trying to execute plan with unsupported UF");
+  ++LoopsVectorized;
```

Comment:
Would be good to leave the other increments as-is for now. 

Doesn't this change now increments LoopsVectorized twice per loop if the epilogue gets vectorized?